README.md

```diff
@@ -100,3 +100,26 @@ end
 ## Signature
 
 `Castle::SecureMode.signature(user_id)` will create a signed user_id.
+
+## Async tracking
+
+By default Castle sends requests synchronously. To eg. use Sidekiq to send requests in a background worker you can pass data to the worker:
```

Comment:
By default Castle sends tracking requests synchronously. To use Sidekiq to send requests to a background worker you can pass data to the worker:

Reply:
should we use 'tracking' word here ? Fine with removing 'eg'. Why `in` and not `to` ?